xbmc/video/VideoDatabase.cpp

```diff
@@ -4602,11 +4607,40 @@ void CVideoDatabase::UpdateTables(int iVersion)
     m_pDS->exec("ALTER TABLE tvshow ADD userrating integer");
     m_pDS->exec("ALTER TABLE musicvideo ADD userrating integer");
   }
+
+  if (iVersion < 97)
+  {
+    // Add idSeason to episode table, so we don't have to join via idShow and season in the future
+    m_pDS->exec("ALTER TABLE episode ADD idSeason integer");
+
+    std::vector<std::tuple<int, int, int>> seasons;
+    m_pDS->query("select idSeason, idShow, season from seasons");
+    while (!m_pDS->eof())
+    {
+      seasons.push_back(std::make_tuple(m_pDS->fv(0).get_asInt(), m_pDS->fv(1).get_asInt(), m_pDS->fv(2).get_asInt()));
```

Comment:
You could use `m_pDS2` here to execute the update statement directly instead of going through a list and do it all later.